### PR TITLE
[forumone/generator-web-starter#88] Add Drupal 8 Capistrano config

### PR DESCRIPTION
**Pull request by fafnirical**
Fixes forumone/generator-web-starter#88, relates to forumone/web-starter#216.

Please review the Ruby syntax, as I don't know Ruby. 😕

### CODE REVIEW

File: web-starter/templates/lib/capistrano/tasks/drush.rake [hunk 1]
```diff
@@ -5,31 +5,37 @@
...
     set :drupal_features_path, %w[]
+    set :drupal_cmi, false
     set :drupal_db_updates, true
     set :settings_file_perms, '644'
     set :site_directory_perms, '750'
   end
 end
 
 namespace :drush do
+  if fetch(:platform) == "drupal8"
```
Comment (blakethomp) [inline web-starter/templates/lib/capistrano/tasks/drush.rake:35]: This only worked for me when set in the defaults, after setting directory permissions. Also the platform seems to be set in web.rake back to "drupal" and not what is set in deploy.rb

File: web-starter/templates/lib/capistrano/tasks/drush.rake [hunk 2]
```diff
@@ -127,12 +133,25 @@ namespace :drush do
     end
   end
 
-  desc "Clears the Drupal cache"
-  task :cc do
-    on roles(:db) do
-      within "#{release_path}/#{fetch(:app_webroot, 'public')}" do
-        fetch(:site_url).each do |site|
-          execute :drush, "-y -p -r #{current_path}/#{fetch(:webroot, 'public')} -l #{site}", 'cc all'
+  if fetch(:platform) == "drupal8"
```
Comment (blakethomp) [inline web-starter/templates/lib/capistrano/tasks/drush.rake]: Tasks defined within a conditional don't seem to be found, removing the conditional around drush:cr/cc and drush:configuration:import/sync worked, relying on the conditional around invoking.

Comment (wwhurley) [reply web-starter/templates/lib/capistrano/tasks/drush.rake]: I think that's correct, we don't need to conditionals around the task configuration.

File: web-starter/templates/lib/capistrano/tasks/drush.rake (hunk 1 above)
Comment (wwhurley) [reply web-starter/templates/lib/capistrano/tasks/drush.rake:35]: Which defaults are you referring to, @blakestein?

Comment (blakethomp) [reply web-starter/templates/lib/capistrano/tasks/drush.rake:35]: Line 25-30 in drush.rake. I had to move 
`if fetch(:platform) == "drupal8"
      SSHKit.config.command_map[:drush] = "../vendor/drush/drush/drush"
end`
to line 31 to get it to be set when running these commands.

File: web-starter/templates/lib/capistrano/tasks/drush.rake (hunk 2 above)
Comment (fafnirical) [reply web-starter/templates/lib/capistrano/tasks/drush.rake]: Okay, I'll work on fixing that.

Comment (fafnirical) [reply web-starter/templates/lib/capistrano/tasks/drush.rake]: Updated!

**Comment by wwhurley** [comment]
Resolved by forumone/generator-web-starter-capistrano#9

**Pull request closed without merging**